boilerplate/fullstack/remix/app/routes/_index.tsx

```diff
@@ -27,8 +25,7 @@ export async function loader({ request }: LoaderFunctionArgs): Promise<{
     const { session, hasInvalidClaims, hasToken, RemixResponse } = await getSessionDetails(request);
 
     const res: SessionProps = {
-        userId: session?.getUserId(),
-        sessionHandle: session?.getHandle(),
+        userId: session?.getUserId() as string,
```

Comment:
not a good idea to use `as string` cause session can be undefined. Instead, make `res: SessionProps | undefined`. Also, change the variable name `res` to something better